algorithms/indonesian.sbl

```diff
@@ -62,7 +62,7 @@ backwardmode (
     // prefix not in {ke, peng, per}
     define SUFFIX_KAN_OK as (
         // On page 29, the example "kompas Q.31" says "Both Nazief and Porter
-        // stemmer converted the word peledakan (blast, explotion) to ledak (to
+        // stemmer converted the word peledakan (blast, explosion) to ledak (to
```

Comment:
I'm pretty sure that this is meant to be an English word...

Reply:
Me too, but this is a verbatim quote from the paper describing the algorithm we're implementing and I think it's best overall to quote exactly what is written in the paper since we're quoting it here to note an inconsistency between two parts of that paper and how we've resolved that in our implementation.

We could add a "(sic)" or similar.

Reply:
Fair. Sold.